modules/lambdas/main.tf

```diff
@@ -199,52 +199,31 @@ resource "aws_lambda_function" "copy_files_to_archive" {
       DATABASE_USER         = var.database_app_user
       COPY_RETRIES          = var.orca_recovery_retry_limit
       COPY_RETRY_SLEEP_SECS = var.orca_recovery_retry_interval
+	    DB_QUEUE_URL          = var.orca_sqs_status_update_queue_id
```

Comment:
Can remove the following since they would not be needed anymore.

PREFIX                 = var.prefix
DATABASE_PORT   = var.database_port
DATABASE_NAME  = var.database_name
DATABASE_USER   = var.database_app_user

Spacing seems odd on new variables added, may want to check.

Reply:
Removed